pygsti/circuits/circuit.py

```diff
@@ -927,6 +891,33 @@ def __add__(self, x):
 
         return Circuit._fastinit(self.layertup + x.layertup, new_line_labels, editable=False, name='',
                                  stringrep=s, occurrence=None)
+    
+
+    def sandwich(self, x, y):
+        """
+        Method for sandwiching labels around this circuit.
+
+        Parameters
+        ----------
+        x : tuple of `Label` objects
+            Tuple of Labels to prepend to this
+            Circuit.
+        
+        y:  tuple of `Label` objects
+            Same as `x`, but appended instead.
+
+        Returns
+        -------
+        Circuit
+        """
+
+        assert(isinstance(x, tuple) and isinstance(y, tuple)), 'Only tuples of labels are currently supported by `sandwich` method.'
+        combined_sandwich_labels = x + y
+        assert(all([isinstance(l, _Label) for l in combined_sandwich_labels])), "Only Circuits and Label-tuples can be added to Circuits!"
+        new_line_labels = set(sum([l.sslbls for l in combined_sandwich_labels if l.sslbls is not None], 
+                                  self._line_labels)) #trick for concatenating multiple tuples
+        new_line_labels = sorted(list(new_line_labels))
```

Comment:
There's no need to convert to a list before passing into ``sorted``. Examples:
```python
>>> sorted({'c','a','b'})
['a', 'b', 'c']
>>> sorted(('c', 'b', 'a'))
['a', 'b', 'c']
```

Reply:
Fixed with the latest commit, thanks!